src/dist/conf/config.yaml

```diff
@@ -35,6 +35,9 @@ http-tracing:
     enabled: false
     zipkin-url: "http://localhost:9411/api/v2/spans"
 
+#Enable JMX Prometheus Metrics
```

Comment:
`# Enable JMX Prometheus Metrics`

Do we want a `jmx-metrics` section with an `enabled: true/false` section anticipating that we will need additional configuration here?